metricbeat/tests/system/test_base.py

```diff
@@ -71,7 +71,7 @@ def test_dashboards(self):
         )
         exit_code = self.run_beat(extra_args=["setup", "--dashboards"])
 
-        assert exit_code == 0
+        assert exit_code == 0, 'Error output: ' + self.get_log()
```

Comment:
Not related, but I guess it doesn't do any harm :slightly_smiling_face:

Reply:
Actually it helps you finding the reason why the `test_dashboards` step has failed in the CI environment ;)